frontend/dispatcher.lua

```diff
@@ -504,7 +579,8 @@ function Dispatcher:addItem(caller, menu, location, settings, section)
                     end,
                     separator = settingsList[k].separator,
                 })
-            elseif settingsList[k].category == "string" then
+            elseif settingsList[k].category == "string"
+            or settingsList[k].category == "configurable" then
```

Comment:
May indent the "or" line (or make it all a single line), a bit hard to read as it is here. There's also one above at line 379.